<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureTrackingParamsClass</class>
 <widget class="QDialog" name="TextureTrackingParamsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Box Tracking Params</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_confirm">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Video Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_video_read_path"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>关键点数量上限:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_numKeyPoints"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Yml Path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_yml_read_path"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>RANSAC最大迭代数:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_iterationsCount"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ply Path:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_ply_read_path"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>卡尔曼滤波最小内点数:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_minInliersKalman"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>保存路径:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_saveDirectory"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ratio:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEdit_ratioTest"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>特征描述方法:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_featureName">
        <item>
         <property name="text">
          <string>ORB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KAZE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AKAZE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BRISK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SIFT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SURF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BINBOOST</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VGG</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>RANSAC重投影误差:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEdit_reprojectionError"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>PnP方法:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_pnpMethod">
        <item>
         <property name="text">
          <string>ITERATIVE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EPNP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P3P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DLS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AP3P</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>RANSAC置信度:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEdit_confidence"/>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Intrinsic</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_fx"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>fx:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>fy:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>cy:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="lineEdit_fy"/>
         </item>
         <item row="5" column="4">
          <widget class="QLineEdit" name="lineEdit_cy"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>cx:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="lineEdit_cx"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox_useFLANN">
           <property name="text">
            <string>FLANN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_fast_match">
           <property name="text">
            <string>Fast Match</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_displayFilteredPose">
           <property name="text">
            <string>display filtered pose</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_confirm</sender>
   <signal>clicked()</signal>
   <receiver>TextureTrackingParamsClass</receiver>
   <slot>onPush_confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>TextureTrackingParamsClass</receiver>
   <slot>onPush_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPush_confirm()</slot>
  <slot>onPush_cancel()</slot>
 </slots>
</ui>
